library/SteamworksPy.cpp

```diff
@@ -13,6 +13,7 @@
 #include "TargetConditionals.h"
 #define SW_PY extern "C" __attribute__ ((visibility("default")))
 #elif defined( __linux__ )
+#include <cstdint>
```

Comment:
Does not compile on my machine without this include:

```
SteamworksPy.cpp:90:14: error: ‘int32_t’ in namespace ‘std’ does not name a type
   90 |         std::int32_t result;
      |              ^~~~~~~
SteamworksPy.cpp:91:14: error: ‘uint64_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   91 |         std::uint64_t publishedFileId;
      |              ^~~~~~~~
      |              wint_t
SteamworksPy.cpp: In member function ‘void Workshop::OnItemSubscribed(RemoteStorageSubscribePublishedFileResult_t*, bool)’:
SteamworksPy.cpp:185:112: error: too many initializers for ‘SubscriptionResult’
  185 |             SubscriptionResult result{itemSubscribedResult->m_eResult, itemSubscribedResult->m_nPublishedFileId};
      |                                                                                                                ^
SteamworksPy.cpp: In member function ‘void Workshop::OnItemUnsubscribed(RemoteStorageUnsubscribePublishedFileResult_t*, bool)’:
SteamworksPy.cpp:192:116: error: too many initializers for ‘SubscriptionResult’
  192 |             SubscriptionResult result{itemUnsubscribedResult->m_eResult, itemUnsubscribedResult->m_nPublishedFileId};
      |                                                                                                                    ^
make: *** [Makefile:2: SteamworksPy.so] Error 1
```

I use Arch and gcc 13.1.1